src/EPR.ProducerContentValidation.Application/Validators/GroupedValidators/ErrorValidators/ConsistentDataSubmissionPeriodsGroupedValidator.cs

```diff
@@ -16,12 +16,12 @@ public ConsistentDataSubmissionPeriodsGroupedValidator(IMapper mapper, IIssueCou
         _issueCountService = issueCountService;
     }
 
-    public override async Task ValidateAsync(List<ProducerRow> producerRows, string storeKey, string blobName, List<ProducerValidationEventIssueRequest> errorRows, List<ProducerValidationEventIssueRequest>? warningRows = null)
+    public override async Task ValidateAsync(List<ProducerRow> producerRows, string storeKey, string blobName, List<ProducerValidationEventIssueRequest> errorRows = null, List<ProducerValidationEventIssueRequest>? warningRows = null)
     {
         var remainingErrorCount = await _issueCountService.GetRemainingIssueCapacityAsync(storeKey);
 
-        var firstProducerRow = producerRows.First();
-        var inconsistentDataSubmissionPeriodRow = producerRows.FirstOrDefault(x => x.DataSubmissionPeriod != firstProducerRow.DataSubmissionPeriod);
+        var firstProducerRow = producerRows[0];
```

Comment:
I think we need to handle null here. Null check?

Reply:
Done.